qa/L0_lifecycle/test.sh

```diff
@@ -1636,8 +1636,8 @@ mkdir models
 cp -r ../python_models/cuda_memory_consumer models/cuda_memory_consumer_1 && \
     cp -r ../python_models/cuda_memory_consumer models/cuda_memory_consumer_2
 
-# Run server to stop model loading if > 30% of GPU 0 memeory is used
-SERVER_ARGS="--model-repository=`pwd`/models --model-control-mode=explicit --model-load-gpu-limit 0:0.3"
+# Run server to stop model loading if > 60% of GPU 0 memeory is used
+SERVER_ARGS="--model-repository=`pwd`/models --model-control-mode=explicit --model-load-gpu-limit 0:0.6"
```

Comment:
Maybe add simple negative tests for the checks you added expecting server to fail?

i.e.
```
SERVER_ARGS="--model-repository=`pwd`/models --model-control-mode=explicit --model-load-gpu-limit 0:<invalid_fraction>
```
and
```
SERVER_ARGS="--model-repository=`pwd`/models --model-control-mode=explicit --model-load-gpu-limit <negative_device_id>:0.6
```